cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java

```diff
@@ -64,28 +64,38 @@
     rules.add(new Rule(ImmutableList.of(Constants.Metrics.Tag.CLUSTER_METRICS)));
     // app, prg type, prg name
     rules.add(new Rule(ImmutableList.of(Constants.Metrics.Tag.APP, Constants.Metrics.Tag.PROGRAM_TYPE,
-                                        Constants.Metrics.Tag.PROGRAM)));
+                                        Constants.Metrics.Tag.PROGRAM),
+                       Constants.Metrics.Tag.DATASET));
```

Comment:
The alignment is correct. Line 68 is the second argument of the ``Rule`` constructor. It is not part of the ``ImmutableList``, which is the first argument to the ``Rule`` constructor.

Agreed that its hard to notice though :)